app/app.go

```diff
@@ -103,6 +109,7 @@ type GaiaApp struct {
 	paramsKeeper   params.Keeper
 	upgradeKeeper  upgrade.Keeper
 	evidenceKeeper evidence.Keeper
+	ibcKeeper      ibc.Keeper
```

Comment:
Transfer keeper also needs to be added here.

Reply:
Help pushing changes appreciated!